### PR TITLE
Update CI image versions to latest

### PR DESCRIPTION
**Pull request by aBozowski**
#### Problem
What is being fixed? 
* Update image versions in CI:
  * GCB Configs are not using the latest image version
  * Workflow configs are not using the latest image version

#### Change overview
* New script to update image versions across CI to latest
* Update GCB Configs to use latest image version
* Update workflow configs to use latest image version

### CODE REVIEW

File: scripts/update_ci_docker_images_to_latest.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+GCB_DIR = 'integrations/cloudbuild/'
+VERSION_FILE = 'integrations/docker/images/chip-build/version'
+
+def get_version(version_file: str) -> str:
+    """Reads the provided images version file and returns the current images version."""
+    with open(version_file) as f:
+        l = f.readline()
+        return l[:l.index(' ')]
+
+def update_line(version: str, line: str, end_quote=False) -> str:
```
Comment (cpagravel) [inline scripts/update_ci_docker_images_to_latest.py:31]: end_quote: bool = False

File: scripts/update_ci_docker_images_to_latest.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    """Replaces the end of a string after the last colon with the specified version.
+    end_quote ends the line with a closing double quote.
+    """
+    if not end_quote:
+        return line[:line.rindex(':')+1]+version
+    else:
+        return line[:line.rindex(':')+1]+version+'"'
+
+def update_file(version: str, file: str, search_term: str, end_quote=False) -> None:
```
Comment (cpagravel) [inline scripts/update_ci_docker_images_to_latest.py:40]: end_quote: bool = False

**Review by cpagravel — APPROVED** [review]
Type hinting looks good.

**Review by bzbarsky-apple — APPROVED** [review]

File: .github/workflows/docker_img.yaml [hunk 3]
```diff
@@ -69,5 +69,5 @@ jobs:
               uses: crazy-max/docker-scan-action@master
               with:
                   # NOTE: This task validates the images built previously with latest tag
-                  image: connectedhomeip/chip-build${{ matrix.img }}:latest
+                  image: connectedhomeip/chip-build${{ matrix.img }}:0.5.70
```
Comment (Arkq) [inline .github/workflows/docker_img.yaml:72]: I think that the version here should be "latest", so the CI will verify lastly build Docker image.

**Pull request closed without merging**